lib/smart_answer_flows/benefit-cap-calculator/questions/enter_postcode.govspeak.erb

```diff
@@ -1,8 +1,12 @@
 <% content_for :title do %>
-  Enter your postcode:
+  Enter your postcode
 
 <% end %>
 
+<% content_for :body do %>
+  eg SW1A 2AA
+<% end %>
+
```

Comment:
Ideally the changes in this commit would've been split up a bit more with the corresponding changes to the regression test artefacts included in each commit.